day2/day2.js

```diff
@@ -0,0 +1,54 @@
+// (() => { anon arrow function
+// })();
+//var { input } = require(`./input.js`);
+
+const convertInput = (input) => {
+  let command = input.split('\n ');
+  arr = [];
+  command.forEach((element) => {
+    let output = element.split(' ');
+    let obj = {};
+
+    obj.direction = output[0];
+    obj.distance = parseInt(output[1]);
+    arr.push(obj);
+  });
+  return arr;
+};
+
+// use global variables without breaking encapsulation
+function main() {
+  let input = `forward 5
```

Comment:
I can't see you use your input.js file. Did you mean to pass that into convertInput?